python/tank/deploy/tank_commands/setup_project_core.py

```diff
@@ -187,7 +187,7 @@ def _project_setup_internal(log, sg, sg_app_store, sg_app_store_script_user, set
     
     try:
         fh = open(roots_path, "wt")
-        yaml.dump(roots_data, fh)
+        yaml.safe_dump(roots_data, fh)
```

Comment:
Same comment about safe_dump vs dump. I think we should actually document the code why it is better to use it. To me safe_dump sounds like it might be a bit slower and someone who wants to optimize yaml parsing in the future might look at this and think "maybe we should change this". Documenting why we have safe_dump at the call site would make the intent clearer I think.